docusaurus/docs/dev-docs/migration/v4-to-v5/additional-resources/plugins-migration.md

```diff
@@ -1,35 +1,50 @@
 ---
-title: Plugins upgrade summary
+title: Plugins upgrade
 displayed_sidebar: devDocsMigrationV5Sidebar
 tags:
   - plugins
   - plugins development
   - upgrade to Strapi 5
 ---
 
-# Plugins upgrade summary
+# Plugins upgrade
 
 The present page is intended to be used as a short summary of everything to consider if you are a plugin developer intending to upgrade your plugin from Strapi v4 to Strapi 5. The page quickly describes the changes affecting plugins and links to additional resources where necessary.
 
+:::caution
+Starting with Strapi 5, the platform includes a [Plugin SDK](https://github.com/strapi/sdk-plugin) to help develop and build Strapi plugins. While you are free to use other methods for plugin development, we strongly recommend using the Plugin SDK. As part of the migration process from v4 to v5, we encourage you to update your plugin to take advantage of the Plugin SDK.
```

Comment:
```suggestion
Starting with Strapi 5, the platform includes a [Plugin SDK](/dev-docs/plugins/development/plugin-sdk) to help develop and build Strapi plugins. While you are free to use other methods for plugin development, we strongly recommend using the Plugin SDK. As part of the migration process from v4 to v5, we encourage you to update your plugin to take advantage of the Plugin SDK.
```

The Plugin SDK docs page already links to GitHub and I'd rather we guide users to docs when they are already inside docs, whenever possible 😊